{{cookiecutter.project_slug}}/static/src/scss/_variables.scss

```diff
@@ -4,73 +4,13 @@ $base-line-height: 1.5 !default;
 $base-font: 'Open Sans';
 
 // Colors
```

Comment:
if it was just and always would be colours, yes!! but there's a few other variables in there like `$width-container` and fonts stuffs! And i think this can be a generic variables file that will expand per project, rather than having a whole variables folder with partials for width, colours etc. Up for debate though, could easily be better to have them separate later if we're using more variables

Reply:
Cool, makes sense!